src/presenter/trip-event-presenter.js

```diff
@@ -1,67 +1,60 @@
-import { render, replace } from '../framework/render.js';
-import ListEvent from '../view/event-list-view.js';
-import EventsItemView from '../view/events-item-view.js';
-import EventEdit from '../view/event-edit.js';
+import { render } from '../framework/render.js';
 import ListEmpty from '../view/list-empty.js';
-export default class EventPresenter {
+import PointPresenter from './point-presenter.js';
+import EventsListView from '../view/events-list-view.js';
 
-  #eventComponent = new ListEvent();
+export default class EventPresenter {
+  #eventsListView = new EventsListView();
   #points = null;
   #listContainer = null;
   #point = null;
   #offer = null;
   #destination = null;
 
+  #pointPresenter = null;
+
   constructor({listContainer, pointsModel, offersModel, destinationsModel}){
     this.#listContainer = listContainer;
     this.#point = pointsModel;
     this.#offer = offersModel;
     this.#destination = destinationsModel;
-    this.#points = pointsModel.get();
+    this.#points = new Map();
   }
 
```

Comment:
Добавляем метод обработки изменения точки: принимаем обновленные данные из дочернего презентера и отправляем их в модель для обновления, а после находим этот презентер и запускаем у него заново метод init чтобы перерисовать точку уже с новыми данными
```suggestion
#handlePointChange = (updatedPoint) => {
    this.#pointsModel.updatePoint(updatedPoint);
    this.#points.get(updatedPoint.id)?.init(updatedPoint)
  };
```